webapp/STYLE_GUIDE.md

```diff
@@ -0,0 +1,157 @@
+# Webapp Style Guide
+
+## Network & Data Layer
```

Comment:
I'm going to reorder these categories a bit. I'd like to call out the things that are the most likely for people to run into first and then go into things that are less common for people to run into after. I'm thinking of focusing on React stuff first and then Redux and then going from there